nservicebus/hosting/critical-errors.md

```diff
@@ -44,14 +44,29 @@ Next define what action to take when this scenario occurs:
 snippet:CustomHostErrorHandlingAction
 
 
-## When to override the default action
+## When to override the default critical error action
 
-The default action should be overridden whenever that default does not meet the specific hosting requirements. For example
+The default behavior is to stop the endpoint and not the process. It is recommended to exit the process when a critical error occurs. Not overriding the critical error callback will result in the endpoint instance to stop processing messages without the ability to recover from this state.
```

Comment:
> The default behavior is to stop the endpoint and not the process. It is recommended to exit the process when a critical error occurs. Not overriding the critical error callback will result in the endpoint instance to stop processing messages without the ability to recover from this state.

To be honest I'm not sure if I understand this paragraph. Are we essentially saying that the default implementation is broken and you should always override it, otherwise you won't be able to recover? Is that true for all hosts?